app/controllers/API/V1/comments_controller.rb

```diff
@@ -9,6 +10,7 @@ def index
   end
 
   def create
+    authorize! :create, Comment
     @post = Post.find(params[:post_id])
     @comment = @post.comments.new(text: comment_params[:text], user_id: current_user.id)
```

Comment:
- [ ] @Team Great work adding controllers for your API, when I  run the API in Postman I get the error below
![image](https://github.com/Graciano1997/technology_blog_app/assets/61408860/2a69be88-b686-4496-a910-cf816817a3d5)
![image](https://github.com/Graciano1997/technology_blog_app/assets/61408860/e0e5a9c3-512f-4493-954b-016a5689a257)